docs/scripts/generate_examples_tests.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+# Copyright (c) 2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+"""Generates examples/tests.py."""
+
+import os
+from contextlib import redirect_stdout
+from pathlib import Path
+from sys import path
+
+# Override global path to load pyavd from pwd instead of any installed version.
```

Comment:
"to load pyavd"? What is that product?!? :P

Reply:
hehe